src/styles/card.css

```
@@ -0,0 +1,106 @@
+/* #region card */
+.card {
+  box-sizing: border-box;
+  padding: 32px 19px;
+  width: 200px;
+}
+
+.card__image {
+  width: 160px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.card__title {
+  display: flex;
+  text-align: center;
+}
+
+.title {
+  color: #060b35;
+  font-family: "Roboto", serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 10px;
+  line-height: 18px;
+  padding-top: 40px;
+}
+
+.card__product {
+  padding-top: 4px;
+  font-family: "Roboto", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  text-align: right;
+  align-items: flex-end;
+  padding: 16px 0;
+}
+
+.reviews {
+  font-family: "Roboto", serif;

```

```suggestion
  font-family: inherit

```
move `font-family: "Roboto", serif;` to the `html` selector